tests/src/main/java/org/hibernate/beanvalidation/tck/tests/metadata/ParameterDescriptorTest.java

```diff
@@ -39,14 +42,7 @@
 	@Deployment
 	public static WebArchive createTestArchive() {
 		return webArchiveBuilder()
-				.withTestClass( ParameterDescriptorTest.class )
-				.withClasses(
-						Account.class,
-						Customer.class,
-						CustomerService.class,
-						Executables.class,
-						Person.class
-				)
+				.withTestClassPackage( ParameterDescriptorTest.class )
```

Comment:
It's not better. It's just more practical. There isn't any value here excluding classes from the test so the practical side wins IMHO.